### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by FurkanS1821**
- Updated NuGet packages with newer releases available.
- Removed unused library references in all projects.
- Readded the after build command that copies the `Settings` folder.

LeagueSandbox/LeagueSandbox-Default#73 must be merged before this one.

***DO NOT MERGE IMMEDIATELY, SUBREPO POINTER HAS TO BE FIXED.***

### CODE REVIEW

File: GameServerLib/GameServerLib.csproj [hunk 1]
```diff
@@ -387,18 +290,22 @@
...
+  </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <PropertyGroup>
     <PostBuildEvent>
     </PostBuildEvent>
   </PropertyGroup>
   <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
     <PropertyGroup>
-      <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
+      <ErrorText>Bu proje bu bilgisayarda olmayan NuGet paketlerine başvuru yapıyor. Bunları indirmek için NuGet Paket Geri Yükleme'yi kullanın. Daha fazla bilgi için, bkz. http://go.microsoft.com/fwlink/?LinkID=322105. Eksik dosya: {0}.</ErrorText>
```
Comment (MythicManiac) [inline GameServerLib/GameServerLib.csproj:304]: English please

File: GameServerLib/GameServerLib.csproj [hunk 2]
```diff
@@ -387,18 +290,22 @@
...
     </PropertyGroup>
-    <Error Condition="!Exists('..\packages\Microsoft.Net.Compilers.2.8.2\build\Microsoft.Net.Compilers.props')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\Microsoft.Net.Compilers.2.8.2\build\Microsoft.Net.Compilers.props'))" />
+    <Error Condition="!Exists('..\packages\NETStandard.Library.2.0.3\build\netstandard2.0\NETStandard.Library.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\NETStandard.Library.2.0.3\build\netstandard2.0\NETStandard.Library.targets'))" />
```
Comment (MythicManiac) [inline GameServerLib/GameServerLib.csproj:306]: ...why did this get downgraded?

Comment (FurkanS1821) [reply GameServerLib/GameServerLib.csproj:306]: 🤔 VS fucked something up I think, let me check

**Review by MatthewFrench — APPROVED** [review]

**Comment by FurkanS1821** [comment]
I see that very same PR was also made by @MatthewFrench. I will not waste my time trying to find what causes Appveyor to fail.

**Pull request closed without merging**